pep-0703.rst

```diff
@@ -0,0 +1,1691 @@
+PEP: 703
+Title: Making the Global Interpreter Lock Optional in CPython
+Author: Sam Gross <colesbury at gmail.com>
+Sponsor: Łukasz Langa <lukasz at python.org>
+Discussions-To: Discourse
```

Comment:
> Note that the "Discussions-To" field is a placeholder. I'll update the field once the PEP is an official draft and the corresponding thread is created on http://discuss.python.org/.

The linter needs it to be a link or empty. We can fill it after merging and when the thread is opened.

```suggestion
Discussions-To:
```